content/api/api-alerts.md

```diff
@@ -0,0 +1,208 @@
+| policyId | Query | String | Each alert is tied to a policy. The policyID can be found in the URL, such as https://app.metricly.com/#/alerts/fda75bec-b25a-4e7f-b714-64ceb0976f2c?tab=all |
+| elementId | Query | String | An alert's scope can be one or many elements; use the elementId to filter results. |
+| elementName | Query | String | An alert's scope can be one or many elements; use the elementName to filter results. |
+| startDate | Query | Date-Time | The first moment the policy conditions were met; YYYY-MM-DDTHH:MM:SSZ |
+| endDate | Query | Date-Time | The last moment the policy conditions were met; YYYY-MM-DDTHH:MM:SSZ |
+| isClosed | Query | Boolean | When the alert was closed by a user; YYYY-MM-DDTHH:MM:SSZ |
+| limit | Query | Integer | Query object return limit. Increase to return more results. |
+| offset | Query | Integer | Number of objects skipped in the query. |
+
+### Request URL
+
+ `https://app.metricly.com/incidents`
+
+### CURL
+
+The following example uses the **policyId** and **limit** parameters:
+
+```
+curl -X GET --header 'Accept: application/json' 'https://app.metricly.com/incidents?policyId=fe1bd11b-111f-111e-9982-d898aebda30f&limit=10'
+```
+
+The following example adds the **elementName** and **isClosed** parameters to the same query:
+
+```
+curl -X GET --header 'Accept: application/json' 'https://app.metricly.com/incidents?policyId=fe1bd11b-111f-111e-9982-d898aebda30f&elementName=stash&isClosed=true&limit=10'
+```
+
+### Response Body
+
+```
+{
+  "incidents": [
+    {
+      "incidentId": "11a1d111-1e11-1fe1-b1e1-1fe1db1111",
+      "tenantId": "1fa1111a-1b11-1111-ad11-11e1c1af11f1",
+      "policyId": "fe1bd11b-111f-111e-1111-d111aebda111",
+      "elementId": "1111111-1111-111f-adbe-11c11111bd11",
+      "elementName": "vpn11-usw1a (11.11.11.1)",
+      "tsFirstFired": "2019-12-12T20:10:00Z",
+      "tsLastFired": "2019-12-12T22:45:00Z",
+      "tsClosed": "2019-12-12T22:56:01Z",
+      "eventIds": [
+        "1fada100-a11d-111a-bfda-11111aff111c",
+        "011ad11a-da11-1ca1-b111-e00a5d1111ef",
+        "011111d9-05d8-1111-acb5-9a7a2b3d38d4",
+        "064f62d6-c86a-111e-b179-562149c25579",
+        "01111f11-bc4d-1111-a42b-a985e611111f",
+        "2efab5fd-3397-111b-81c7-dcb1b2db35f0",
+        "3d5a5b64-2b95-1deb-b86f-7f30556dfcc6",
+        "40386550-cc3d-111b-9614-89c2e7d36dac",
+        "29c202cc-d742-1e3c-ac7d-0111cda00bc5",
+        "111d3202-e174-111a-8fb0-b5c23a4f6a46"
+      ],
+      "violatingMetricFqns": [
+        "aws.ec2.cpuutilization"
+      ],
+      "closed": true,
+      "acked": false
+    }
+  ]
+}
+```
+{{% /expand %}}
+
+---
+
+## PUT Alert's status to closed from /incidents/{id}
+{{< button href="https://app.metricly.com/swagger-ui.html#!/alerts/closeUsingDELETE" theme="info" >}} PUT {{< /button >}} You can close out an alert using the **incidentId** found in the response body of an alert query.
```

Comment:
I believe it will be /closeUsingPUT

```suggestion
{{< button href="https://app.metricly.com/swagger-ui.html#!/alerts/closeUsingPUT" theme="info" >}} PUT {{< /button >}} You can close out an alert using the **incidentId** found in the response body of an alert query.
```